apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs

```diff
@@ -0,0 +1,339 @@
+            // Create client
+            BigtableClient bigtableClient = BigtableClient.Create();
+            // Initialize request argument(s)
+            TableName tableName = new TableName("[PROJECT]", "[INSTANCE]", "[TABLE]");
+            RowKey rowKey = new RowKey(12, 255, 0, 17, 1);
+            ReadModifyWriteRule rule1 = ReadModifyWriteRules.Append("cf1", "c1", "_original");
+            ReadModifyWriteRule rule2 = ReadModifyWriteRules.Increment("cf1", "c2", 1);
+            // Make the request
+            ReadModifyWriteRowResponse response = bigtableClient.ReadModifyWriteRow(
+                tableName,
+                rowKey,
+                rule1, rule2);
+            Console.WriteLine($"Row key: {response.Row.Key.ToStringUtf8()}");
+            foreach (Family family in response.Row.Families)
+            {
+                Console.WriteLine($"  Family name: {family.Name}");
+                foreach (Column column in family.Columns)
+                {
+                    Console.WriteLine($"    Column qualifier: {column.Qualifier.ToStringUtf8()}");
+                    foreach (Cell cell in column.Cells)
+                    {
+                        Console.WriteLine($"      New cell value: {cell.Value.ToStringUtf8()}");
+                        Console.WriteLine($"      Timestamp: {cell.Version.ToDateTime()}");
+                        Console.WriteLine($"      Labels: {string.Join(", ", cell.Labels)}");
+                    }
+                }
+            }
+            // End snippet
+
+            // TODO: Verifications
+        }
+
+        public async Task ReadRows()
+        {
+            // Snippet: ReadRows(TableName,RowSet,RowFilter,long,CallSettings)
+            // Create client
+            BigtableClient bigtableClient = BigtableClient.Create();
+            // Initialize request argument(s)
+            TableName tableName = new TableName("[PROJECT]", "[INSTANCE]", "[TABLE]");
+            RowSet rows = RowSet.FromRowRanges(
+                RowRange.Open("a", "z"),
+                RowRange.Closed(null, new RowKey(9, 128, 4)));
+            RowFilter filter = RowFilters.Chain(
+                RowFilters.FamilyNameRegex("A"),
+                RowFilters.Interleave(
+                    RowFilters.PassAllFilter(),
+                    RowFilters.Chain(
+                        RowFilters.ApplyLabelTransformer("foo"),
+                        RowFilters.Sink())),
+                RowFilters.ColumnQualifierRegex("B"));
+            long rowsLimit = 10;
+            // Make the request
+            BigtableClient.ReadRowsStream streamingResponse = bigtableClient.ReadRows(
+                tableName,
+                rows,
+                filter,
+                rowsLimit);
+            // Read streaming responses from server until complete
+            IAsyncEnumerator<Row> responseStream = streamingResponse.GetRowsAsync().GetEnumerator();
+            while (await responseStream.MoveNext())
```

Comment:
Ok, I'll add that here. Also, now that I think of it, rather than adding a `Cancel()` to `ReadRowsStream`, it probably makes more sense for `GetRowsAsync()` to take an optional CancellationToken which it monitors for cancellations and then disposes the underlying gRPC call when is detects cancellation. I'm going to change things to do that, but let me know if you feel otherwise.